package.json

```diff
@@ -41,13 +41,13 @@
     "cheerio": "1.0.0-rc.11",
     "css-url-parser": "^1.0.0",
     "debug": "^4.3.1",
-    "fs-extra": "^10.0.0",
     "got": "^12.0.0",
     "lodash": "^4.17.21",
     "normalize-url": "^7.0.2",
     "p-queue": "^7.1.0",
     "sanitize-filename": "^1.6.3",
-    "srcset": "^5.0.0"
+    "srcset": "^5.0.0",
+    "zlib": "^1.0.5"
```

Comment:
Why are we using `zlib` from npm instead of [built-in node.js module](https://nodejs.org/docs/latest-v14.x/api/zlib.html)?

Reply:
We actually did a bunch of testing on this a couple of years ago to best make use of our redis storage capacity. We determined that zlib overall had the best overall balance between compression ratio and compress/decompress performance at various data sizes. High performance read/write are needed and the data sizes are very similar to our use case internally so I used it on that basis.

Reply:
On [npm](https://www.npmjs.com/package/zlib) I can see that last version was published 11 years ago.
Also [github repo for this package](https://github.com/kkaefer/DEPRECATED-node-zlib) says it's deprecated
> This extension is deprecated since the functionality was folded into node.js core: https://nodejs.org/dist/latest/docs/api/zlib.html

So for me it looks like built-in node module would be better here in order to avoid usage of deprecated packages.